docs/Config_Reference.md

```diff
@@ -318,6 +317,83 @@ radius:
 #   just prior to starting a probe operation. The default is 5.
 ```
 
+### Deltesian Kinematics
+
+See [example-deltesian.cfg](../config/example-deltesian.cfg) for an
+example linear delta kinematics config file. See the
+[delta calibrate guide](Delta_Calibrate.md) for information on
+calibration.
+
+Only parameters specific to deltesian printers are described here - see
+[common kinematic settings](#common-kinematic-settings) for available
+ parameters.
+
+```
+[printer]
+kinematics: deltesian
+max_z_velocity:
+#   For deltesian printers, this limits the maximum velocity (in mm/s) of
+#   moves with z axis movement. This setting can be used to reduce the
+#   maximum speed of up/down moves (which require a higher step rate
+#   than other moves on a deltesian printer). The default is to use
+#   max_velocity for max_z_velocity.
+#max_z_accel:
+#   This sets the maximum acceleration (in mm/s^2) of movement along
+#   the z axis. Setting this may be useful if the printer can reach higher
+#   acceleration on XY moves than Z moves (eg, when using input shaper).
+#   The default is to use max_accel for max_z_accel.
+#minimum_z_position: 0
+#   The minimum Z position that the user may command the head to move
+#   to. The default is 0.
+#min_angle: 5
+#   This represents the minimum angle (in degrees) relative to horizontal
+#   that the deltesian arms are allowed to achieve. This parameter is
+#   intended to restrict the arms from becomming completely horizontal,
+#   which would risk accidental inversion of the XZ axis. The default is 5.
+#print_width:
```

Comment:
There isn't really an origin associated with this configuration option. This is the minimum angle the arms are allowed to be at versus horizontal (with 0 degrees being completely horizontal, and 90 degrees being completely vertical).

I agree this could be clearer in the description, though.

Reply:
My comment was about `print_width` parameter though and not `min_angle`. The latter is explained pretty well, at least, in my opinion.

Reply:
Ah, sorry, my mistake. The code snippet came through kinda weird.

`print_width` is the complete width of the printable area.

e.g. `print_width: 200` results in the X axis being able to move from -100 to 100.

Reply:
Having our heads in the kinematics for a while makes it obvious to us, so an outside view is always interesting. Does adding this sentence at the end of the comment clarify the situation in your opinion?
> If specified, the origin (0,0) is in the middle of print_width value.

Reply:
I guess my main confusion (perhaps it is confusion?) is related to the fact that the user can specify different `arm_x_length` for left and right tower. Does that shift the origin? I mean, `position_endstop` technically references 'the center of the build area and the endstops are triggered' and `arm_x_length` 'when the printer is homed' (all 2 endstops for left and right tower are triggered?). And for a delta printer, the situation is a bit different because there is a single `delta_radius` parameter that defines an origin and the towers are equidistantly placed from it, and `print_radius` just references the same origin.

So, perhaps your suggestion to add 'If specified, the origin (0,0) is in the middle of print_width value' makes sense. Or maybe 'The print area over X axis is [-print_width/2, print_width/2]', which also references the coordinate origin. Not sure if there is any need to clarify the origin placement.